neuralprophet/configure.py

```diff
@@ -140,6 +126,39 @@ def __post_init__(self):
         if len(self.quantiles) > 1:
             self.loss_func = PinballLoss(loss_func=self.loss_func, quantiles=self.quantiles)
 
+    def set_quantiles(self):
+        # assert either prediction interval or quantiles is None, or both are None
+        assert self.prediction_interval is None or self.quantiles is None, "Prediction interval and quantiles cannot both be populated, one or both must be None."
+        if self.uncertainty_method.lower() in ["quantile_regression", "quantile regression", "qr"]:
+            # assert prediction interval is None and quantiles is a list
+            assert self.prediction_interval is None and isinstance(self.quantiles, list), "Prediction interval must be None while quantiles must be a list. Otherwise , set uncertainty estimation param to 'auto'"
+        elif self.uncertainty_method.lower() in ["conformal_prediction", "conformal prediction", "cp"]:
+            # assert prediction interval is a float and quantiles is None
+            assert isinstance(self.prediction_interval, float) and self.quantiles is None, "Prediction interval must be a float while quantiles must be None. Otherwise , set uncertainty estimation param to 'auto'"
+        elif self.uncertainty_method.lower() in ["auto", "a"]:
+            # assert that prediction interval is a float between (0, 1) if not None, then use that to create the quantiles
+            if self.prediction_interval is not None:
+                assert isinstance(self.prediction_interval, float) and (0 < self.prediction_interval < 1), \
+                    "The prediction interval specified needs to be a float in-between (0, 1)."
+                alpha = 1 - self.prediction_interval
+                self.quantiles = [alpha/2, 1 - alpha/2]
+        else:
+            raise ValueError("The only valid uncertainty_method options are 'auto', 'quantile_regression', or 'conformal_prediction'.")
```

Comment:
Let's leave conformal_prediction out of the message until it's available

Reply:
Done